apps/page_mapping/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ config_option(
     DEFAULT
     ON
     DEPENDS
-    "DefaultBenchDeps"
+    "DefaultBenchDeps; KernelSel4ArchAarch64 OR KernelArchX86"
```

Comment:
What about RISC-V now actually?

Reply:
Yeah, I was thinking that too. Since RISC-V uses pretty much the same kind of page table structure as AARCH64, it would probably work with these changes. Do you think that's worth adding to this PR?

Reply:
I you have the time to check RISC-V also, that would be good. Seems we are just excluding arm32 then.

Reply:
Finally had some time to revisit this, I have confirmed the benchmarks work on RISC-V (though I haven't run the full benchmark with up to 2048 pages and 16 runs per configuration to completion as it takes forever).